src/Utils.ts

```diff
@@ -182,7 +182,7 @@ export const deployTransactionRequestShape = {
   },
   metadata: {
     relayHubAddress: ow.string,
-    relayMaxNonce: ow.number,
+    relayMaxNonce: ow.string,
```

Comment:
Why did we change this to string? If they are numbers, I'd keep it as a number.

Reply:
sorry I though that this number definition was coming from typechain, but its not. updated